weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -996,7 +996,7 @@ def feedback_create(self, req: tsi.FeedbackCreateReq) -> tsi.FeedbackCreateRes:
             "wb_user_id": req.wb_user_id,
             "creator": req.creator,
             "feedback_type": req.feedback_type,
-            "payload": payload,
+            "payload": req.payload,
```

Comment:
i see... the ORM does the encoding. Nice. 

idea: can the ORM enforce MAX_PAYLOAD to avoid double stringification?

Reply:
Yes, in some databases I would expect the database itself to have the length constraint on the column, but I don't think that's the ClickHouse way, so doing it at our ORM level makes sense. I can add that as a follow on. Since we do have quite a strict size limit, the second encoding is expected to be fast, but it would be better to only do it once.